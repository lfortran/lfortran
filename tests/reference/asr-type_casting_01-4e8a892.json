{
    "basename": "asr-type_casting_01-4e8a892",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/type_casting_01.f90",
    "infile_hash": "9385ba3715f01c20aaa23ba05b0a20383ed149344bc72e7ede478970",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-type_casting_01-4e8a892.stderr",
    "stderr_hash": "372b13217c84710fd3f0d667e77c8a2f2516da635a765fedc1858a4d",
    "returncode": 2
}