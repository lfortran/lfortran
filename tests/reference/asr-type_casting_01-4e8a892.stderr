semantic error: Type mismatch in array initialization.
 Enable logical casting by setting `--logical-casting = true`
 --> tests/errors/type_casting_01.f90:3:5
  |
3 |     logical :: x(3) = [1, 1, 0]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
